Expecto/Expect.fs

```diff
@@ -434,13 +434,13 @@ let isFasterThanSub (f1:Performance.Measurer<_,_>->'a) (f2:Performance.Measurer<
     Tests.failtestf "%s. Expected f1 (%s) to be faster than f2 (%s) but is ~%.0f%% slower."
                     format (toString s1) (toString s2) ((s1.mean/s2.mean-1.0)*100.0)
   | Performance.MetricLessThan (s1,s2) ->
-    Impl.logger.info (
+    Impl.logger.log Info (
```

Comment:
If we expect this to be sent all the way to the terminal before the process exists, `logWithAck` should be used.